utils/lock/utils_windows.go

```diff
@@ -5,14 +5,14 @@ import (
 	"syscall"
 )
 
-// This file will be compiled on windows.
+// This file will be compiled on Windows.
 // Checks if the process can be reached.
 // If an error occurs, check if the error is part of the invalid parameter. This means the process is not running.
 // Else find the exit code. If the exit code 259 means the process is running.
 func isProcessRunning(pid int) (bool, error) {
 	process, err := syscall.OpenProcess(syscall.PROCESS_QUERY_INFORMATION, true, uint32(pid))
 	if err != nil {
-		// Check if err is type of syscall.Errno that is windows error number.
+		// Check if err is type of syscall.Errno that is Windows error number.
```

Comment:
// Check if err is type of syscall.Errno that is Windows error number.
-->
// Check if err is of type of syscall.Errno, which is a Windows error number.